### PR TITLE
Generate smart account and sponsor userOperation

### PR DESCRIPTION
**Pull request by JohnGuilding**
* Deploys Kernel account via permissionless
* Calls hats contract to test account deployment and sponsorship

### CODE REVIEW

File: packages/interface/src/config.ts [hunk 1]
```diff
@@ -68,6 +73,19 @@ export const getRPCURL = (): string | undefined => {
...
 };
 
+/**
+ * Get the PIMLICO RPC URL based on the network we are connected to
+ * @returns the PIMLICO RPC URL
+ */
+export const getPimlicoRPCURL = (): string | undefined => {
+  switch (process.env.NEXT_PUBLIC_CHAIN_NAME) {
+    case "optimismSepolia":
+      return `https://api.pimlico.io/v2/optimism-sepolia/rpc?apikey=${process.env.NEXT_PUBLIC_PIMLICO_API_KEY}`;
```
Comment (ctrlc03) [inline packages/interface/src/config.ts:83]: aren't endpoints using the chainId? i.e. `https://api.pimlico.io/v2/10/rpc?apiKey`?

File: packages/interface/src/pages/signup/registerEmail.tsx [hunk 2]
```diff
@@ -81,7 +121,14 @@ const RegisterEmail = (): JSX.Element => {
 
   const generateEmbeddedAccount = async () => {
     console.log("Generating new account");
-    return "0x91AdDB0E8443C83bAf2aDa6B8157B38f814F0bcC"; // TODO: (merge-ok) hardcoded address will be swapped for generated account address in next PR
+    const privateKey = generatePrivateKey();
```
Comment (ctrlc03) [inline packages/interface/src/pages/signup/registerEmail.tsx:124]: let's save it to localstorage?

File: packages/interface/src/config.ts (hunk 1 above)
Comment (JohnGuilding) [reply packages/interface/src/config.ts:83]: Didn't know it works with chainId too. Looks like it works with both https://docs.pimlico.io/permissionless/how-to/accounts/use-kernel-account#create-the-clients

File: packages/interface/src/pages/signup/registerEmail.tsx (hunk 2 above)
Comment (JohnGuilding) [reply packages/interface/src/pages/signup/registerEmail.tsx:124]: done

**Review by ctrlc03 — APPROVED** [review]
LGTM thanks

**Pull request merged**